### ISSUE TITLE
[Accessibility : Focus & Sequence] Focus is missing for following frameset items

### ISSUE THREAD

**Issue by Nagamani71** [kimchi-project/kimchi#917]
![accessibility26](https://cloud.githubusercontent.com/assets/17590096/14145470/a23a2000-f6b1-11e5-9247-51a45b785b01.PNG)
